README.md

```diff
@@ -12,43 +12,73 @@
  License.
 -->
 
-# Google Cloud Platform: github-actions
+# `setup-gcloud` GitHub Action
 
-This repository contains a library of [Github Actions](https://github.com/actions) providing functionality for working with [Google Cloud Platform](http://cloud.google.com/).
+Configures the [Google Cloud SDK][sdk] in the GitHub Actions environment. The Google Cloud SDK includes both the [gcloud][gcloud] and
+[gsutil][gsutil] binaries.
 
-## Available Actions
+Find other Google Cloud GitHub Actions in [`google-github-actions`](https://github.com/google-github-actions).
 
-* [setup-gcloud](./setup-gcloud/README.md): This action downloads, installs, and configures a [gcloud Cloud SDK](https://cloud.google.com/sdk/) environment for the worker, adding the `gcloud` CLI command to the worker's $PATH.
+**This repository also contains deprecated GCP GitHub Actions from previous mono-repo format.**
 
-* [get-secretmanager-secrets](./get-secretmanager-secrets/README.md): This action accesses secrets from [Google Secret Manager](https://cloud.google.com/secret-manager) and makes their results available as output variables.
+## Table of Contents
 
-## Example Workflows
+* [Usage](#usage)
+* [Inputs](#inputs)
+* [Example Workflows](#example-workflows)
+
+## Usage
+
+```yaml
+- name: Set up Cloud SDK
+  uses: google-github-actions/setup-gcloud@master
+  with:
+    version: '290.0.1'
+    project_id: ${{ secrets.GCP_PROJECT_ID }}
+    service_account_key: ${{ secrets.GCP_SA_KEY }}
+    export_default_credentials: true
+
+- name: Use gcloud CLI
+  run: gcloud info
+```
 
-* [Google Kubernetes Engine](./example-workflows/gke/README.md): An example workflow that uses [GitHub Actions][github-action] to deploy a static website to an existing [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) cluster.
+## Inputs
 
-* [Cloud Run](./example-workflows/cloud-run/README.md): An example workflow that uses [GitHub Actions][github-action] to build and deploy a container to [Cloud Run](https://cloud.google.com/run/).
+| Name          | Requirement | Default | Description |
```

Comment:
This is awesome, we should do it for all the other actions. Is there a tool for doing this?

Reply:
Good! I thought this was more readable. I just copied and pasted into a markdown table, but there are tools like https://tableconvert.com/

Reply:
gotcha, I am thinking of writing a small script that can do this for use. I will update in a PR.